migration_original/ODS1Stage/tables/Base/ClientFeatureToClientFeatureValue/spu_translated_ClientFeatureToClientFeatureValue.sql

```diff
@@ -0,0 +1,565 @@
+CREATE OR REPLACE PROCEDURE ODS1_STAGE.BASE.SP_LOAD_CLIENTFEATURETOCLIENTFEATUREVALUE() -- Parameters
+    RETURNS STRING
+    LANGUAGE SQL
+    EXECUTE AS CALLER
+    AS  
+DECLARE 
+---------------------------------------------------------
+--------------- 0. Table dependencies -------------------
+---------------------------------------------------------
+    
+--- BASE.CLIENTFEATURE
+--- BASE.CLIENTFEATUREVALUE
+--- BASE.SWIMLANE_BASE_CLIENT
+
+---------------------------------------------------------
+--------------- 1. Declaring variables ------------------
+---------------------------------------------------------
+
+    select_statement STRING; -- CTE and Select statement for the Merge
+    update_statement STRING; -- Update statement for the Merge
+    insert_statement STRING; -- Insert statement for the Merge
+    merge_statement STRING; -- Merge statement to final table
+    status STRING; -- Status monitoring
+   
+---------------------------------------------------------
+--------------- 2.Conditionals if any -------------------
+---------------------------------------------------------   
+
+BEGIN
+
+---------------------------------------------------------
+----------------- 3. SQL Statements ---------------------
+---------------------------------------------------------     
+
+--- Select Statement
+-- If no conditionals:
+select_statement := $$with cte_swimlane as (
+    select
+        *,
+        rank() over(
+            partition by customerproductcode
+            order by
+                LastUpdateDate
+        ) as rowrank,
+    from
+        base.swimlane_base_client
+),
+cte_tmp_features as (
+    select
+        CustomerProductCode,
+        'FCBFN' as ClientFeatureCode,
+        FeatureFCBFN as ClientFeatureValueCode
+    from
+        cte_swimlane
+    where
+        FeatureFCBFN = 'FVNO'
+        and RowRank = 1
+    union
```

Comment:
Question, could these "union" be "union all "? Usually, "union all" has better performance.